### PR TITLE
Falco to build on aarch64

### LINKED ISSUE #520: ARM64 Build
Hey,

Can you guys release ARM64 builds? I mainly run Kubernetes on ~50-100 ARM clusters and wanted to test Falco out. I did try compiling myself, but gRPC was throwing an odd error during compilation at me (I fixed the errors thrown by protobuf by providing my own compiled installation of protobuf on the system). I assume I could maybe compile newer grpc and provide it on the system... but I'd rather not hack together a build process to get a build for a single release.

Many thanks in advance.

**What would you like to be added**:
Deb, RPM and pre-built packages for ARM7 and ARM8.

**Why is this needed**:
We want to have packages for other platforms than X86_64 - ARM7 and ARM7 both seem platforms on which many users want to try Falco.

**Related issues**
- #586

### PR DESCRIPTION
**Pull request by fntlnz**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area build

**What this PR does / why we need it**:

What happens here:

- [x] libscap, libsinsp and drivers updated to the `fntlnz-aarch64` branch, this is because we needed to add some preprocessor gates for aarch64. **This needs to be changed with the specific commit id and SHA before merging** https://github.com/draios/sysdig/pull/1701
- [x] luajit updated to git tag `1d8b747c161db457e032a023ebbff511f5de5ec2` - this is because luajit didn't tag since 2017, but they are actively developing it. Initially we wanted to switch to moonjit to support also ppc64le but the new luajit commits also address that along with the 64 bit changes needed for arm64. Please
- [x] gRPC updated to 1.32.0 [read more](https://github.com/falcosecurity/falco/pull/1442/commits/358851de1132e9495ddd037394124d75bace1ceb)
- [x] cpack fixes to produce aarch64 packees
- CI build targets - opened an issue for this #1445 

**Which issue(s) this PR fixes**:

Fixes #520 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
build: updated LuaJIT to 1d8b747c161db457e032a023ebbff511f5de5ec2
build: updated gRPC to v1.32.0
build: updated sinsp, scap and the drivers to 5c0b863ddade7a45568c0ac97d037422c9efb750
build: use fields_info from libsinsp
build: updated b64 to ce864b17ea0e24a91e77c7dd3eb2d1ac4175b3f0
```

### CODE REVIEW

**Comment by fntlnz** [comment]
This also supersedes the changes made for ppc64le https://github.com/falcosecurity/falco/pull/1225 since the only change needed here was to use moonjit instead of luajit. Many projects and OSes (like [Alpine](https://pkgs.alpinelinux.org/package/edge/main/x86/luajit-dev)) already use moonjit as a synonim of luajit.

This will likely require some testing but the next release is on December 1st so there's no better time than now.

**Comment by fntlnz** [comment]
This PR also updates libsinsp and libscap, it's important to note that those already depend on moonjit for certain architectures and that there was a breaking change that needed to be addressed: https://github.com/draios/sysdig/pull/1693

**Comment by fntlnz** [comment]
Note: 2.1.2 seems to be the latest moonjit that works for our lua code.

Tried to update to 2.2.0 and we start having issues. I know we want to start making plans to remove lua as we already did for the outputs in #1412 so I'll just note this here and we can follow up on that if we don't make such plan in the near future.

```
---
- required_engine_version: 7
# Currently disabled as read/write are ignored syscalls. The nearly
# similar open_write/open_read check for files being opened for
# reading/writing.
# - macro: write
#   condition: (syscall.type=write and fd.type in (file, directory))
# - macro: read
#   condition: (syscall.type=read and evt.dir=> and fd.type in (file, directory))
---. Exiting.
```

**Comment by fntlnz** [comment]
I switched back to Luajit but got the latest master. See details in the commit https://github.com/falcosecurity/falco/pull/1442/commits/0a08a045ad9730ac88661cce7b372278e40b6f1c

**Review by leodido — DISMISSED** [review]

**Comment by fntlnz** [comment]
The minimal build fails are related to our sinsp dependency. Working on it https://github.com/draios/sysdig/pull/1703

File: cmake/modules/sysdig.cmake
Comment (leogr) [inline cmake/modules/sysdig.cmake:32]: LGTM, once that has been updated

**Comment by fntlnz** [comment]
/hold cancel

Ready for review

**Review by leogr — APPROVED** [review]
Just tried on an aarch64 instance with 18.04.5 LTS (Bionic Beaver).
It works nicely! :clap:

File: cmake/modules/sysdig.cmake
Comment (leodido) [inline cmake/modules/sysdig.cmake:60]: 💯  finally!

File: userspace/falco/CMakeLists.txt
Comment (leodido) [inline userspace/falco/CMakeLists.txt:74]: Is this duplicate? See line 69

```suggestion
```

File: userspace/falco/CMakeLists.txt
Comment (leodido) [inline userspace/falco/CMakeLists.txt:84]: Cool approach

**Review by leodido — APPROVED** [review]
Great job! 👏 

Left a suggestion only

**Comment by leodido** [comment]
/hold

just for the suggestion

File: userspace/falco/CMakeLists.txt
Comment (fntlnz) [reply userspace/falco/CMakeLists.txt:74]: yes thanks!

**Pull request merged**
